docs/nuxt.config.js

```diff
@@ -1,15 +1,15 @@
 import { withDocus } from 'docus'
 
 export default withDocus({
-  docus: {
-    colors: {
-      primary: '#00DC82'
+  rootDir: __dirname,
+  socialImage: {
+    baseUrl: 'DOMIN',
+    chrome: {
+      browserWSEndpoint: 'wss://chrome.browserless.io/'
```

Comment:
Is there any reason why docus is not setting this for default? (PS not sure but `nuxt-extend` should set `rootDir` from top layer if not we can investigate)

Reply:
@Tahul

Reply:
Agree this should be set by default

Reply:
Resolved by https://github.com/nuxtlabs/docus/commit/bee93deced36a04f5b3471e6ffa37352400cccaf